app/Jobs/ApiHrmsSyncJob.php

```
@@ -0,0 +1,50 @@
+        $this->method = $method;
+    }
+
+    /**
+     * Execute the job.
+     */
+    public function handle(): void
+    {
+        try {
+            $service = app(HrmsService::class);
+            if (!method_exists($service, $this->method)) {
+                Log::warning("ApiHrmsSyncJob: Method {$this->method} does not exist.");
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'handle'. (Clean Code Rules)

(StaticAccess)